core/test/helpers/swap.ts

```diff
@@ -0,0 +1,75 @@
+import getSigner from '../../src/signer';
```

Comment:
I actually think it's fine to include this functionality to the core itself (as we might even provide this feature later on to the user). Just please be more specific with naming ABIs, eg `UNISWAP.json` -> `UNISWAP_V2_ROUTER_02.json` (using the contract name)